Lib/ssl.py

```diff
@@ -894,7 +894,10 @@ def write(self, data):
 
         The 'data' argument must support the buffer interface.
         """
-        return self._sslobj.write(data)
+        if data != b'':
+            return self._sslobj.write(data)
+        else:
+            return ""
```

Comment:
``write`` returns the number of bytes sent. The function should either return ``0`` or fail with a more useful exception.